content/docs/cml/self-hosted-runners.md

```diff
@@ -79,31 +79,31 @@ We like the
+| `--name <string>`                 | `cml-{identifier}` where `{identifier}` is a random string | Name displayed in the repository once registered                                                                                                      |
+| `--single`                        | N/A                                                        | Exit after running a single job                                                                                                                       |
+| `--reuse`                         | N/A                                                        | Don't launch a new runner if an existing one has the same name or overlapping labels                                                                  |
+| `--driver <platform>`             | Inferred from environment variables                        | Platform where the repository is hosted. Accepts `github` or `gitlab`                                                                                 |
+| `--repo <URL>`                    | Inferred from environment variables                        | Git repository URL e.g. https://github.com/iterative/cml                                                                                              |
+| `--token <token>`                 | Inferred from environment variables                        | Personal access token with enough permissions to register a self-hosted runner on the repository                                                      |
+| `--cloud <vendor>`                | N/A                                                        | Cloud provider to deploy the runner. Accepts `aws` or `azure`. If not set, the runner will be deployed locally and `--cloud-` options will be ignored |
+| `--cloud-region <region>`         | `us-west`                                                  | Region where the instance is deployed. Accepts `us-east`, `us-west`, `eu-west`, `eu-north` or native regions                                          |
+| `--cloud-type <type>`             | N/A                                                        | Instance type. Accepts `m`, `l`, `xl` or native types like `t2.micro`                                                                                 |
+| `--cloud-gpu <type>`              | `nogpu`                                                    | GPU type. Accepts `nogpu`, `k80` or `tesla`.                                                                                                          |
+| `--cloud-hdd-size <size>`         | N/A                                                        | Storage size in GB                                                                                                                                    |
```

Comment:
Maybe
```suggestion
| `--cloud-hdd-size <gigs>`         | N/A                                                        | Storage size in GB                                                                                                                                    |
```

Reply:
@jorgeorpinel, what about `gigabytes` without abbreviations?

`<kidding>`
Mostly to avoid triggering my obsessive-compulsive disorder of expanding abbreviations whenever possible and in every context. In my humble opinion, and as far as I know, they only look good to me when the original words are preserved.
`</kidding>`

Reply:
> what about gigabytes?

Sure
```suggestion
| `--cloud-hdd-size <gigabytes>`    | N/A                                                        | Storage size in GB                                                                                                                                    |
```